maintenance_product/views/maintenance_equipment_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="pm_maintenance_equipment_form_view" model="ir.ui.view">
+        <field name="name">pm.maintenance.equipment.form.view</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="arch" type="xml">
+            <field name="category_id" position="after">
+                <field name="product_category_id" invisible="1" />
+                <field name="product_id" />
```

Comment:
```suggestion
                <field name="product_id" context="{'default_maintenance_ok': True}" />
```
It would be nice to add context in the form field, so if you create a product on the fly it will meet the domain requirements. Or just disable the create option.
For the rest LGTM

Reply:
Thank you for the review. I followed your suggestion and implemented that feature.